src/main/java/com/ning/maven/plugins/dependencyversionscheck/AbstractDependencyVersionsMojo.java

```diff
@@ -241,6 +245,18 @@
     /** Keeps track of the longest name for an artifact for printing out nicely. */
     protected int maxLen = -1;
 
+    private final ExecutorService executorService = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 5, new ThreadFactory() {
```

Comment:
You can add it (this plugin is old enough to be originally written before Guava existed)